platform/viewer/src/index.js

```diff
@@ -15,9 +15,10 @@ import ReactDOM from 'react-dom';
  * to the window for processing.
  */
 import loadDynamicImports, { loadRuntimeImports } from './pluginImports.js';
+import loadDynamicConfig from './loadDynamicConfig';
 
-loadDynamicImports().then(() => {
-  loadRuntimeImports(window.config).then(() => {
+Promise.all([loadDynamicImports(), loadDynamicConfig(window.config)]).then(
```

Comment:
The dynamic config must get loaded first, because it might define dynamic imports to use.

Reply:
We can waterfall this promise in ~then~